integration_tests/setup_test.go

```diff
@@ -346,14 +363,27 @@ func (s *IntegrationTestSuite) initGenesis() {
 		InitialUnitPriceInUsomm:    sdk.MustNewDecFromStr("2"),
 		CurrentUnitPriceInUsomm:    sdk.MustNewDecFromStr("2"),
 		RemainingTokensForSale:     sdk.NewCoin("gravity0x3506424f91fd33084466f402d5d97f05f8e3b4af", sdk.NewInt(5000000000)),
-		FundingModuleAccount:       cellarfees.ModuleName,
-		ProceedsModuleAccount:      cellarfees.ModuleName,
+		FundingModuleAccount:       cellarfeestypes.ModuleName,
+		ProceedsModuleAccount:      cellarfeestypes.ModuleName,
 	})
 
 	bz, err = cdc.MarshalJSON(&auctionGenState)
 	s.Require().NoError(err)
 	appGenState[auctiontypes.ModuleName] = bz
 
+	// set cellarfees gen state
+	cellarfeesGenState := cellarfeestypes.DefaultGenesisState()
+	s.Require().NoError(cdc.UnmarshalJSON(appGenState[cellarfeestypes.ModuleName], &cellarfeesGenState))
+	cellarfeesGenState.Params = cellarfeestypes.Params{
+		FeeAccrualAuctionThreshold: 2,
+		RewardEmissionPeriod:       100,
+		InitialPriceDecreaseRate:   sdk.MustNewDecFromStr("0.05"),
+		PriceDecreaseBlockInterval: uint64(1000),
```

Comment:
Isn't 1000 blocks a huge interval for an integration test that must run in realtime?

Reply:
yes, seems kind of over the top. I copied this from the auction module genesis.

Reply:
Does this imply the auction integration test isn't checking to see that the price decreases over the block interval?

Reply:
Seems so. I thought about that but didn't make it a concern for this PR.